### PR TITLE
Feature/implements more reviews button

### PR DESCRIPTION
**Pull request by julia-thea**
Implemented More Reviews button. Each time it's clicked, 2 more reviews are loaded at a time until the limit is reached.

@acdavitt @dylanreid7 @ChrisRPeterson

### CODE REVIEW

File: client/components/RatingsAndReviews/Reviews/AddReviewDialog.jsx [hunk 1]
```diff
@@ -43,11 +43,12 @@ const AddReviewDialog = ({ productId }) => {
...
 
-  const classes = useStyles();
+  // const classes = useStyles();
 
   return (
-    <div className={classes.buttons}>
-      <Button variant='contained' color='primary'>MORE REVIEWS</Button>
+    // <div className={classes.buttons}>
+          <div>
+      {/* <Button variant='contained' color='primary'>MORE REVIEWS</Button> */}
```
Comment (dylanreid7) [inline client/components/RatingsAndReviews/Reviews/AddReviewDialog.jsx:51]: Consider deleting all the comments in this file.

File: client/components/RatingsAndReviews/Reviews/Review.jsx [hunk 2]
```diff
@@ -63,6 +63,9 @@ const Review = ({ review }) => {
         <Grid item xs={12}>
           <Divider className={classes.dividerLine} />
         </Grid>
+        <Grid item xs={12}>
+          {/* for spacing purposes */}
```
Comment (dylanreid7) [inline client/components/RatingsAndReviews/Reviews/Review.jsx:67]: Delete comment.

File: client/components/RatingsAndReviews/Reviews/ReviewsList.jsx [hunk 3]
```diff
@@ -20,27 +16,40 @@ const useStyles = makeStyles((theme) => ({
     overflow: 'auto',
   },
   gridList: {
-    minHeight: 200,
-    maxHeight: 500
+    minHeight: 550,
+    maxHeight: 550
```
Comment (dylanreid7) [inline client/components/RatingsAndReviews/Reviews/ReviewsList.jsx:20]: Seems odd to me to have the minHeight and maxHeight be the same value.  Would it be possible to just state height: 500?  Or something similar?

File: client/components/RatingsAndReviews/Reviews/ReviewsList.jsx [hunk 4]
```diff
@@ -20,27 +16,40 @@ const useStyles = makeStyles((theme) => ({
...
   },
   header: {
     fontWeight: 500,
   },
 }))
 
+const getReviewsData = async (id) => {
+      const response = await axios.get(`https://app-hrsei-api.herokuapp.com/api/fec2/hratx/reviews?product_id=${id}&count=100`);
+      return response.data;
```
Comment (dylanreid7) [inline client/components/RatingsAndReviews/Reviews/ReviewsList.jsx:29]: Consider revising spacing.

**Review by dylanreid7 — COMMENTED** [review]
Looks good Julia!  It seems you've properly added the feature that allows for the reviews count to update by 2 each time the user hits the More Reviews button.
A couple of thoughts for potential improvements (it helps that I had a super similar feature in my widget, otherwise I probably wouldn't have noticed :)
- Could be nice to have the More Reviews button disappear or turn into something like "Collapse Reviews" once all reviews are displayed.
- When a user switches products, the number of reviews isn't set back to 2.  I think this is totally ok, but if you want it to reset to 2, it's worth thinking about.
- It could be nice to play with the width of the reviews viewer so that it's not a scrollbar, given that the width is like 95% filled up.

File: client/components/RatingsAndReviews/Reviews/ReviewsList.jsx (hunk 3 above)
Comment (julia-thea) [reply client/components/RatingsAndReviews/Reviews/ReviewsList.jsx:20]: Good point!

File: client/components/RatingsAndReviews/Reviews/ReviewsList.jsx (hunk 4 above)
Comment (julia-thea) [reply client/components/RatingsAndReviews/Reviews/ReviewsList.jsx:29]: 👁 of an 🦅, thanks @dylanreid7!!

File: client/components/RatingsAndReviews/Reviews/Review.jsx (hunk 2 above)
Comment (julia-thea) [reply client/components/RatingsAndReviews/Reviews/Review.jsx:67]: This comment explains to someone reading the code, why there is an empty Grid item here 😆 Removing it messes up the alignment and I'm not sure how else to add spacing between two grid items. 
Also commenting the code to improve it's readability I think is okay

File: client/components/RatingsAndReviews/Reviews/AddReviewDialog.jsx (hunk 1 above)
Comment (julia-thea) [reply client/components/RatingsAndReviews/Reviews/AddReviewDialog.jsx]: Thank you, removing

**Comment by julia-thea** [comment]
@dylanreid7 Thank you for all your comments and suggestions!
I also didn't like the review list being horizontally scrollable, thanks for the idea of setting maxWidth to 95%. It resolved the issue.
The idea of making the button disappear once there are no more reviews makes me internally cringe. I feel like I would be slightly shocked as a user if page elements where disappearing 😅
Changing button text to Less Reviews when all the reviews have been displayed sounds like a really good idea. Could be a good feature to implement in the future. If I get to it today/tomorrow, I'll open a separate PR for it.

Please take another look and let me know if there is anything else I need to address or improve.

File: client/components/RatingsAndReviews/Reviews/Review.jsx (hunk 2 above)
Comment (dylanreid7) [reply client/components/RatingsAndReviews/Reviews/Review.jsx:68]: Sounds good!

**Review by dylanreid7 — APPROVED** [review]

**Pull request merged**